crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -516,6 +522,12 @@ impl TransferAction {
             fee_asset,
         })
     }
+
+    /// Returns action type as `&str` for fee event logging
+    #[must_use]
+    pub fn get_action_type(&self) -> &str {
```

Comment:
As mentioned above. Don't define this an instead rely on `prost::Name::full_name()`